exampleFolder/index.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+var app = express();
+const axios = require('axios');
+const { response } = require('express');
+
+//POST Route (/addPerson)
+app.post('/addPerson', (req, res) => {
+    console.log(req.body)
+    const body = req.body
+
+    axios.post('http://java-sample-api-2020.herokuapp.com/addPerson',body)
+    .then((res) => {
+        console.log(res.data)
+        res.status(200).json(res.data)
+    });
+
+    //Asynchronous Error Handling
+    Promise.resolve().then(() => {
+        throw new Error('Broken')
+    }).catch(next)
```

Comment:
Im not sure if you need the promise.resolve here. Another thing is that line 19 only throws an error and does not send anything back to the user so this request will hang forever.

Reply:
> Im not sure if you need the promise.resolve here. Another thing is that line 19 only throws an error and does not send anything back to the user so this request will hang forever. Same on the other error handlers as well

Changed the error handling functions to .catch() methods.